scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#268.sql

```diff
@@ -0,0 +1,70 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- 1. Clean up animal_release table by deleting animal_release records with no corresponding surgery
+DELETE FROM animal_release ar
+WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
```

Comment:
Under the current application design, there is no way to roll back these types of changes. If we get something wrong, then the original (meta)data is unrecoverable and there is no way unwind to the previous state and reapply change. Are @astecken and @smancini still happy to proceed understanding this caveat?